.editorconfig

```diff
@@ -12,3 +12,7 @@ indent_style = tab
 [*.json]
 indent_style = space
 indent_size = 2
+
+[*.yml]
```

Comment:
The workflow files seem to be formatted using 2-spaces of indentation, so I've added this exception to the editorconfig. Alternatively, I can remove this and indent the workflows with tabs instead.